lib/index.d.ts

```diff
@@ -927,6 +927,16 @@ declare namespace Stardog {
          */
         function setRoles(conn: Connection, username: string, roles: string[], params?: object): Promise<HTTP.Body>;
 
+        /**
+         * Adds a role to a user.
+         * 
+         * @param {Connection} conn the Stardog server connection
+         * @param {string} username the username of the user
+         * @param {string} role the role to add to the user
+         * @param {object} params additional parameters if needed
+         */
+        function addRole(conn: Connection, username: string, role: string, params?: object): Promise<HTTP.Body>;
```

Comment:
what do you think about renaming this `assignRole` to match our current naming convention for `user.assignPermission` and `user.role.assignPermission`?

can you also update the docs by running the `docs` script?